ironfish-rust/src/keys/mod.rs

```diff
@@ -210,23 +210,22 @@ impl SaplingKey {
         &self.incoming_viewing_key
     }
 
-    /// Retrieve both the view keys. These would normally used for third-party audits
+    /// Retrieve the sapling_representation of the ViewKey. These would normally used for third-party audits
     /// or for light clients.
-    pub fn view_keys(&self) -> ViewKeys {
-        ViewKeys {
-            incoming: self.incoming_view_key().clone(),
-            outgoing: self.outgoing_view_key().clone(),
-        }
-    }
-
-    /// Adapter to convert this key to a viewing key for use in sapling
+    /// -    /// Adapter to convert this key to a viewing key for use in sapling
     /// functions.
```

Comment:
Can you clean up this comment? Think it got clobbered when you deleted the other struct